yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/call/factory/MessageFactory.java

```diff
@@ -0,0 +1,23 @@
+package com.yoti.api.client.spi.remote.call.factory;
+
+import java.io.UnsupportedEncodingException;
+
+import static com.yoti.api.client.spi.remote.Base64.base64;
+import static com.yoti.api.client.spi.remote.call.YotiConstants.DEFAULT_CHARSET;
+
+public class MessageFactory {
```

Comment:
This class seems not to be a Factory class if we mean `Factory` as the Design Pattern. Maybe we can use another name, like MessageGenerator for example.

Reply:
To me, it creates Messages, so it's a MesageFactory.
It is definitely doing some creation.  In this case it returns an Array object instead of a Message object, but MessageFactory is probably a nicer name than ArrayFactory or MessageArrayFactory.